api/v1/users/PKPUserHandler.inc.php

```diff
@@ -24,26 +24,31 @@ class PKPUserHandler extends APIHandler {
 	 */
 	public function __construct() {
 		$this->_handlerPath = 'users';
-		$roles = array(ROLE_ID_SITE_ADMIN, ROLE_ID_MANAGER, ROLE_ID_SUB_EDITOR);
-		$this->_endpoints = array(
-			'GET' => array (
-				array(
+		$roles = [ROLE_ID_SITE_ADMIN, ROLE_ID_MANAGER, ROLE_ID_SUB_EDITOR];
```

Comment:
As these roles already have access to all users, I guess it's ok to keep the same for the report.